pkg/cluster/cluster.go

```diff
@@ -296,6 +299,12 @@ func (c *Cluster) Create() error {
 		c.logger.Infof("databases have been successfully created")
 	}
 
+	if c.Postgresql.Spec.EnableLogicalBackup {
+		if err := c.createBackupCronJob(); err != nil {
+			return fmt.Errorf("could not create a k8s cron job for logical backups: %v", err)
```

Comment:
Should it be a fatal error or is it something we can proceed with?

Reply:
good question ! here I've chosen to follow the creation pattern of all other k8s resources in this method: if a resource cannot be created, exit with error

Reply:
the advantage of the approach is that  a misconfigured backup has more chances to fail fast and hence be immediately visible